state-sync/storage-service/server/src/tests/mock.rs

```diff
@@ -213,14 +213,6 @@ fn get_random_network_id() -> NetworkId {
     }
 }
 
-/// Initializes the Aptos logger for tests
-fn initialize_logger() {
```

Comment:
why move this away? is it going to be referenced somewhere else additionally? Wait for that code change to do this move?

Reply:
Aah, see this: https://github.com/aptos-labs/aptos-core/pull/9104. It's the motivation for most of the moves. The PR is already huge, so I didn't want to bloat it further with util changes 😄